zfixtures/fake-ci-issue-api.json

```
@@ -2,7 +2,7 @@
     {
         "httpRequest": {
             "method": "POST",
-            "path": "/v1/repos/Mergifyio/gha-ci-issues/ci_issues_upload",
+            "path": "/v1/repos/Mergifyio/gha-mergify-ci/ci_issues_upload",
```

```suggestion
            "path": "/v1/repos/Mergifyio/gha-mergify-ci/ci_insights_upload",
```
Can we rename the fixtures also?

---

Last released `mergify-cli` still use `/ci_issues_upload`. Next release will change this to `/ci/traces`